content.js

```diff
@@ -620,13 +621,14 @@ function updateSponsorTimesSubmitting() {
     });
 }
 
+isSubmitLoaded = () => (document.getElementById("startSponsorImage") !== undefined);
```

Comment:
Why is this called `isSubmitLoaded` but reference `startSponsorImage`. Also, don't you mean to check if it is equal to null, not undefined?

Reply:
it is called that because the id startSponsorImage is added when createButton is run
if startSponsorImage is not undefined hopefully it exists

Reply:
it is never undefined, document.getElementByID returns null not undefined

Reply:
But why not check for the submitButton. What does `isSubmitLoaded` mean?

Reply:
It should return true once startSubmit button is added.
it needs to be changed to null not undefined

Reply:
Then why doesn't it check if `submitButton` is not null?

Reply:
@ajayyy it does now